pkg/output/output.go

```diff
@@ -216,3 +221,22 @@ func uniqueNormalizeCertNames(names []string) []string {
 	}
 	return results
 }
+
+// mistmatchedCert returns if cert names does not contain host
+func mistmatchedCert(host string, names []string) bool {
```

Comment:
Instead of having this logic here, we should keep it where we calculate self-contained or expired certificates